subprojects/tronkko-dirent.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = dirent-1.23.2
+
+source_url = https://github.com/tronkko/dirent/archive/1.23.2.tar.gz
```

Comment:
Isn't this unneeded post-f664ced049394e2a5d4300ba526e206df73ec729?

Reply:
`compare/compare.c` seems to include `dirent.h` without a define guard, but then re-includes it below with a guard? This might be unintentional, but I think without that fixed it's still necessary.

Reply:
A couple of notes:
 - This meson setup isn't building `compare`, so ompare` using `dirent.h` isn't relevant. 
 - The second include in `compare/compare.c` is `direct.h` not `dirent.h`. The former  is only required on Win32 (and the `_mkdir` signature is different from POSIX `mkdir`)
  - The utilities are only for development, testing, fuzzing, etc and not expected to be as portable as the library.

Reply:
Oh, whoops. Reading is hard. I'll remove the dirent code.